### PR TITLE
compatibility update: CustomElementRegistry

### PR DESCRIPTION
**Pull request by dlabz**
document.registerElement() is deprecated in favor of customElements.define().
With this update stamp.js works on Safari.

### CODE REVIEW

File: src/stamp.js [hunk 1]
```diff
@@ -19,16 +19,16 @@ var Stamp = Stamp || {};
...
-    document.registerElement(name, {
-      prototype: ep
-    });
+    var constr = function() {
+      this.prototype = Object.create(HTMLElement.prototype);
+      Object.keys(proto).forEach(function(key) {
+        this[key] = proto[key];
+       });
+       this.__ = {};
+       __.context = new ns.Context();
```
Comment (jcgregorio) [inline src/stamp.js]: Shouldn't this be:

   this.__.context = new ns.Context();
   this.__.name = name;

Comment (dlabz) [reply src/stamp.js]: Might be... I didn't look in to it too deeply. Noticed it doesn't work on iOS, switched the function, now it works for me.

Comment (dlabz) [inline src/stamp.js]: P.S.

Comment (dlabz) [reply src/stamp.js]: P.S. 
I've never used .. in my js, nor seen it used. What does it do?

Comment (jcgregorio) [reply src/stamp.js]: OK, that's disturbing, the comment system seems to be eating double underscores, let me try again:
I think those lines of code should be:
```
this.__.context = new ns.Context();
this.__.name = name;
```

Comment (dlabz) [reply src/stamp.js]: oh, right.. 
damn, that kind of stuff is ruining my life... 

let me fix that

**Comment by jcgregorio** [comment]
Thanks!
